tests/golos.emit_tests.cpp

```diff
@@ -90,12 +91,15 @@ BOOST_FIXTURE_TEST_CASE(start_emission_test, golos_emit_tester) try {
     prepare_balances();
     produce_block();
 
+    create_account(cfg::vesting_name);
```

Comment:
I believe that the creation of basic accounts should be done in the constructor. To check the parameter setting with a missing account, you can pass to parameters another missing account.
Please, fix this in another places too.